src/typings/gravity.d.ts

```diff
@@ -8,3 +9,5 @@ interface User {
   type?: string
   name?: string
 }
+
+type UserRoles = "admin" | "team" | "user" | "genomer"
```

Comment:
Is it OK that this isn't the [full list](https://github.com/artsy/gravity/blob/master/config/initializers/_user_roles.rb)?

Reply:
I can go ahead and add these - thanks for this list!

Reply:
But does it _have_ to match, or does this just need to include the subset that Force's code references? I wouldn't want the addition of a new role to become runtime errors.

Reply:
I'm pretty sure we are safe on this but will monitor.